homeassistant/components/fritz/__init__.py

```diff
@@ -60,6 +64,30 @@ def _async_unload(event):
     return True
 
 
+async def async_device_setup(
+    hass: HomeAssistant, entry: ConfigEntry, fritz_tools: FritzBoxTools
+):
+    """Set up a device that is online."""
+    dev_reg = await device_registry.async_get_registry(hass)
+    identifier = (DOMAIN, entry.unique_id)
+    device_entry = dev_reg.async_get_device(identifiers={identifier}, connections=set())
+    if device_entry and entry.entry_id not in device_entry.config_entries:
+        device_entry = None
+
```

Comment:
```suggestion
```

Reply:
You shouldn't need this as it will `get` instead of `create` if it already exists